.github/workflows/cd-storybook.yml

```diff
@@ -29,9 +31,7 @@ jobs:
       - name: Setup Node
         uses: actions/setup-node@v3
         with:
-          node-version: "16"
-          cache-dependency-path: ./app/package-lock.json # or yarn.lock
-          cache: npm # or yarn
```

Comment:
ℹ️ I started to add NPM caching to everywhere but it doesn't actually make a noticeable impact. `npm ci` completes within 20-35 seconds with or without this, so I just removed it for simplicity.